subprojects/android-test/android-log/src/main/kotlin/com/avito/android/log/AndroidLoggerFactory.kt

```diff
@@ -0,0 +1,29 @@
+package com.avito.android.log
+
+import com.avito.android.log.sentry.SentryDestination
+import com.avito.logger.DefaultLogger
+import com.avito.logger.Logger
+import com.avito.logger.LoggerFactory
+import com.avito.logger.handler.CombinedHandler
+import com.avito.logger.handler.DefaultLoggingHandler
+
+class AndroidLoggerFactory(private val sentryDsn: String) : LoggerFactory {
```

Comment:
`sentryDsn` is only one parameter. 
It might be worth to encapsulate it in an object.
Can't we use here`SentryConfig`?

Reply:
i rewrote work with sentry in android part, take a look at https://github.com/avito-tech/avito-android/pull/695/commits/7fa736a5ecb38c2e02e38147adb439e6e30db47b